app/src/main/java/nl/nfi/cellscanner/Database.java

```diff
@@ -33,7 +32,12 @@
     private SQLiteDatabase db;
     private Date previous_date = null;
 
-    public static File getDataPath(Context ctx) {
+    public static String getFileTitle() {
```

Comment:
generateFileTitle is misschien beter; het bestand bestaat nog niet op dit punt als ik het goed zie